spec/hydra/works/services/generic_file/generate/extracted_text_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+describe Hydra::Works::GenerateExtractedText do
+  context 'when the object has no original file' do
+    let(:error_message) { 'object has no content at original_file from which to generate extracted text' }
+    let(:object) { double('object', original_file: nil) }
+    it 'raises an error' do
+      expect(-> { described_class.call(object) }).to raise_error(ArgumentError, error_message)
```

Comment:
`expect { described_class.call(object) }.to raise_error...` is the more common way of doing this.